### DISCUSSION TITLE
Contents of the new LOTUS metadata files

### DISCUSSION THREAD

**Discussion by oolonek** [lotusnprod/lotus-processor#87] [General]
We are thinking about improvements to be made on the following files https://zenodo.org/doi/10.5281/zenodo.5794106.

Currently headers of [230106_frozen_metadata.csv.gz](https://zenodo.org/records/7534071/files/230106_frozen_metadata.csv.gz?download=1) are 

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/pma/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/pma/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">

structure_wikidata | structure_inchikey | structure_inchi | structure_smiles | structure_molecular_formula | structure_exact_mass | structure_xlogp | structure_smiles_2D | structure_cid | structure_nameIupac | structure_nameTraditional | structure_stereocenters_total | structure_stereocenters_unspecified | structure_taxonomy_npclassifier_01pathway | structure_taxonomy_npclassifier_02superclass | structure_taxonomy_npclassifier_03class | structure_taxonomy_classyfire_chemontid | structure_taxonomy_classyfire_01kingdom | structure_taxonomy_classyfire_02superclass | structure_taxonomy_classyfire_03class | structure_taxonomy_classyfire_04directparent | organism_wikidata | organism_name | organism_taxonomy_gbifid | organism_taxonomy_ncbiid | organism_taxonomy_ottid | organism_taxonomy_01domain | organism_taxonomy_02kingdom | organism_taxonomy_03phylum | organism_taxonomy_04class | organism_taxonomy_05order | organism_taxonomy_06family | organism_taxonomy_07tribe | organism_taxonomy_08genus | organism_taxonomy_09species | organism_taxonomy_10varietas | reference_wikidata | reference_doi | manual_validation
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

</body>

</html>

Currently we have discussed of 

- pruning this table down to WD QIDs only for structure, organism, reference and adding other associated WD QIDs

- [ ] Define which ones

- sharing externally (not WD) fetched metadata (e.g. OTL taxa names, NPC and ClassyFire classes) associated to this core table in associated tables

- [ ] Define which metadata we would like.
- [ ] Share script to proceed to enrichment processes.

**Comment by Adafede** [comment]
Related to https://github.com/lotusnprod/lotus-search/issues/53, https://github.com/lotusnprod/lotus-search/issues/6 (amongst others like https://github.com/lotusnprod/lotus-search/issues/104, https://github.com/lotusnprod/lotus-search/issues/110, https://github.com/lotusnprod/lotus-search/issues/50, https://github.com/lotusnprod/lotus-search/issues/10)
